pkgs/applications/version-management/git-and-tools/git-imerge/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "git-imerge-${version}";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    repo = "git-imerge";
+    owner = "mhagger";
+    rev = version;
+    sha256 = "09czjxgjbby54jx1v5m825k87v8g9g374hwv0r6ss48kv1ipvakq";
+  };
```

Comment:
Please use a tarball for released versions such as https://github.com/mhagger/git-imerge/archive/0.7.0.tar.gz